### DISCUSSION TITLE
virtual device for temperatures & eltako

### DISCUSSION THREAD

**Discussion by HotteX** [mak-gitdev/HA_enoceanmqtt#47] [Request]
Cheerio, me again :) 

Now the virtual devices are working, can we add a virtual sensor for temperatures? Especially one for Eltako? 

I think A5-10-06 would be a good start as it has temp and set temp and I have the hardware for testing/debugging.

Betriebsart FHK: EEP: A5-10-06
Lerntelegramm: 0x40300D87
Datentelegramm:
Data_byte0 = 0x0F
Data_byte1 = Isttemperatur 0xFF..0x00
entspricht 0..40°C
Data_byte2 = Solltemperatur
0x00..0xFF entspricht 0..40°C

Tell me if I can prepare stuff, so you have less work implementing it.

**Comment by mak-gitdev** [comment]
Hi @HotteX,

I will have a look and tell you once it is done.
I think there will be two slides to configure the temperature and the setpoint.
Each modification will send a new message. Or maybe a specific button to send the message.
It was my understanding that you wanted to do a kind of Zigbee to EnOcean bridge, right ?
If so, I think you will then have to update these two entities with your Zigbee values in an automation.

**Comment by mak-gitdev** [comment]
Hi @HotteX,

Just pushed it to branch [support-new-devices](https://github.com/mak-gitdev/HA_enoceanmqtt/tree/support-new-eeps).

Please report whether it is ok or not.
